cargo-apk/README.md

```diff
@@ -49,10 +49,10 @@ runtime_libs = "path/to/libs_folder"
 
 # See https://developer.android.com/guide/topics/manifest/uses-sdk-element
 #
-# Defaults to a `min_sdk_version` of 23 and `target_sdk_version` is based on the ndk's default platform.
+# Defaults to a `min_sdk_version` of 23 and `target_sdk_version` of 30 (or lower if the detected NDK doesn't support this).
 [package.metadata.android.sdk]
 min_sdk_version = 16
```

Comment:
should maybe update this one as well, considering we internally limit it to >=23
```suggestion
min_sdk_version = 23
```

Reply:
Yeah we should update this as it represents the default.

Note that we're leaving it at 16 for the examples, for the time being.

Reply:
This was actually an oversight in #197, fwiw :)